src/rust/engine/src/nodes.rs

```diff
@@ -308,6 +310,40 @@ impl WrappedNode for Select {
             })
             .to_boxed()
         }
+        &tasks::Rule::Intrinsic(Intrinsic { product, input })
```

Comment:
If this will have exactly the same behavior as the existing intrinsic, it should probably replace it? That would avoid needing the new Node/intrinsic, etc.

To do so, you'd add an python-level "convert an `ExecuteProcessRequest` into a `MultiPlatformExecuteProcessRequest`" `@rule`, and then remove the old intrinsic.

Reply:
Yeah I agree. Basically [here](https://github.com/pantsbuild/pants/pull/8216/files/0d896eafd4dbd5efebf20a267f2840c728ea5aad#diff-11db197014e1e80c9551e9b61aba6aa0R538) I was lifting simple requests to advanced ones on the rust side, but I suppose it does simplify the code to do it on the python side.